basque/COPYING

```diff
@@ -0,0 +1,16 @@
+voc.txt is a mixture of data licensed as BSD 3-clause (as described in
+../COPYING) and CC BY-SA.
+
+voc.txt was derived in part from a dump of the Euskera(Basque) Wikipedia by the
```

Comment:
Is the above accurate, or is the `basque/voc.txt` file here actually only from Wikipedia data?

The parts about "a mixture of data licensed as BSD 3-clause" and "derived in part" are present for some languages where we've combined another word list with the wikipedia data (e.g. for French we needed a larger list to provide test coverage for a fix so we merged wikipedia data into the existing list).  If the data is only from wikipedia, we should just state that - for example, see `lithuanian/COPYING`.